components/cli/pkg/commands/list_Instances.go

```diff
@@ -25,36 +25,20 @@ import (
 
 	"github.com/olekukonko/tablewriter"
 
+	"github.com/cellery-io/sdk/components/cli/kubernetes"
 	"github.com/cellery-io/sdk/components/cli/pkg/kubectl"
 	"github.com/cellery-io/sdk/components/cli/pkg/util"
 )
 
-func RunListInstances() {
-	displayCellTable()
+func RunListInstances(kubeCli kubernetes.KubeCli) {
+	displayCellTable(kubeCli)
 	displayCompositeTable()
 }
 
-func displayCellTable() {
-	cellData, err := kubectl.GetCells()
-	if err != nil {
-		util.ExitWithErrorMessage("Error getting information of cells", err)
-	}
-	if len(cellData.Items) > 0 {
+func displayCellTable(kubeCli kubernetes.KubeCli) {
+	tableData := GetCellTableData(kubeCli)
```

Comment:
please make private - rename to getCellTableData.

Reply:
Updated to private function.